### DISCUSSION TITLE
jdsimplecontactform support

### DISCUSSION THREAD

**Discussion by SerGun78** [templaza/astroid-framework#351] [Q&A] [answered]
Sonny, joomdev had a wonderfully simple and functional form module
While it works on joomla 4.3.1.
But it needs to be supported for future versions of joomla.
Are you unable to support and update this module?
source here: https://github.com/joomdev/JD-Simple-Contact-Form

**Comment by darekmer** [comment]
that would be great, it's a really great module. Without support this module will disappear from Joomla.

**Comment by sonvnn** [comment, marked as answer]
@SerGun78 @darekmer Thank you for suggestion. As discussed before, All of Joomdev extension will be maintained by Stackideas team. Please help me contact @imarklee or send him a support request on https://stackideas.com/

Thanks & Best Regards,
Sonny

**Comment by darekmer** [comment]
Hi, I sent a question to https://stackideas.com. I searched their site and found the free JoomDev templates are now paid for. I think they have done a disservice to the Joomla community.
Message:
Hello, I wanted to ask where are the free extensions from JoomDev that you were supposed to support and develop? Because as I can see, the free templates from JoomDev have become paid.

**Reply by sonvnn** [reply]
Hi @darekmer 

I'm so sorry to hear that. But as we discussed before, I will just maintain the Astroid Framework. All other Joomdev extensions will be maintained by Stackideas. They have full control over their future and decide extensions are free or paid.

Thanks & Best Regards,
Sonny

**Comment by darekmer** [comment]
Hello, I am not a php programmer. I forked the JoomDev repository and decided to play around with the code a bit. What I did: I removed joomla 3 support, used the new joomla 4 namespaces and it works :), (at least with me). There was an error with php 8, but it is ok.
The repo https://github.com/techjoomla/joomla-3x-to-4x-migration-tools was used as a help. Maybe someone could see what else to add or change, or at least check if it works.
